paddle/scripts/docker/build.sh

```diff
@@ -7,43 +7,21 @@ function abort(){
 
 trap 'abort' 0
 set -e
-if [ ${USE_UBUNTU_MIRROR} == "ON" ]; then
-    sed -i 's#http://archive\.ubuntu\.com/ubuntu/#mirror://mirrors\.ubuntu\.com/mirrors\.txt#g'\
-      /etc/apt/sources.list
-fi
-apt-get update
-apt-get install -y cmake libprotobuf-dev protobuf-compiler git \
-    libgoogle-glog-dev libgflags-dev libatlas-dev libatlas3-base g++ m4 python-pip\
-    python-protobuf python-numpy python-dev swig
 
 if [ ${WITH_GPU} == 'ON' ]; then
   ln -s /usr/lib/x86_64-linux-gnu/libcudnn.so /usr/lib/libcudnn.so
```

Comment:
Got it. Hmm, in order to minimize the change, let me leave both of them there at the right moment. But thank you very much for pointing it out! :)